src/lkl/posix-host.c

```diff
@@ -535,6 +533,7 @@ static void* timer_callback(void* _timer)
     mutex_unlock(&timer->mtx);
 
     lthread_exit(NULL);
+    return NULL;
```

Comment:
This adds dead code. The correct fix for this is to add a no return attribute on the lthread exit prototype.

Reply:
Done, I had to add noreturn to `_lthread_yield` and `_switch` (assembly code) as well.

Reply:
Thanks.  Can you now remove the `return NULL;' (it should be raising a warning about unreachable code)?

Reply:
I did that already. GitHub is showing you the original change without an option to see the current state in this thread...